mk/PcapPlusPlus.mk.dpdk

```diff
@@ -10,4 +18,9 @@ PCAPPP_LIBS_DIR += -L$(RTE_SDK)/build/lib -L/lib64
 PCAPPP_BUILD_FLAGS += -msse -msse2 -msse3 -Wall
 
 # libs
+TMPL = $(shell $(PKGCONF) --libs libdpdk)
```

Comment:
Same here:
1. We can add `2> /dev/null` to suppress errors
2. We can separate into 2 parts:
    ```
    TMPLDIR = $(shell $(PKGCONF) --libs-only-L libdpdk)
    TMPL = $(shell $(PKGCONF) --libs-only-other libdpdk)
    ```